zeebe-cluster/values.yaml

```diff
@@ -38,14 +29,70 @@ service:
     port: 26501  
   internal:
     port: 26502
-resources: 
-  requests:
-    cpu: 500m
-    memory: 1Gi
-  limits:
-    cpu: 1000m
-    memory: 2Gi
-global: 
-  elasticsearch:
-    host: "elasticsearch-master"
-    port: 9200 
+
+resources:  {}
+#   requests:
+#     cpu: 500m
+#     memory: 1Gi
+#   limits:
+#     cpu: 1000m
+#     memory: 2Gi
+
+data:
+  enabled: false
+  volumeClaimTemplate:
+    accessModes: [ "ReadWriteOnce" ]
+    storageClassName: gp2
```

Comment:
@ysaakpr we probably don't want this.. why gp2?

Reply:
This is can be any regular spec for volume claim template. The values are modifiable as per the user, if they have ssd configured as storage class they can. By default, storage class on aws is gp2.

Reply:
@ysaakpr but we cannot have anything AWS specific in the charts.. that needs to be overridden, this needs to be "standard"